express/src/controllers/sign-in.ts

```diff
@@ -129,3 +125,14 @@ export const checkEmailPasswordReset = async function (req: Request, res: Respon
     }
     res.render("check-email-password-reset.njk");
 };
+
+export const resendForgotPassword = async function (req: Request, res: Response) {
+    const emailAddress = req.session.emailAddress;
+    try {
+        const s4: SelfServiceServicesService = await req.app.get("backing-service");
+        await s4.forgotPassword(emailAddress as string);
+    } catch (error) {
+        console.error("ERROR CALLING COGNITO - FORGOT PASSWORD WITH EMAIL", error);
+    }
```

Comment:
This can be refactored into a method now

Reply:
Done.